pkgs/by-name/sc/scrypted/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_20,
+  callPackage,
+  nix-update-script,
+}:
+let
+  npmHooks = callPackage ./hooks.nix { };
+  nodejs = nodejs_20;
+in
+(buildNpmPackage.override { inherit nodejs; }) rec {
+  pname = "scrypted";
+  version = "0.126.0";
+
+  src = fetchFromGitHub {
+    owner = "koush";
+    repo = "${pname}";
+    rev = "v${version}";
```

Comment:
```suggestion
    repo = "scrypted";
    tag = "v${version}";
```

Reply:
repo name and pname are not the same things semantically. For example you can override derivation with custom pname, but obviously repo name should not depend on it. So `repo = pname` is an antipattern, but still widely used.

Reply:
Makes total sense, thank you.